es.js

```diff
@@ -92,7 +92,7 @@ export const es = {
     history: "Historia",
     price: "Precio",
     usd: "USD",
-    txs: "TX",
+    txs: "impuestos",
```

Comment:
This is an abbreviation for transactions

Reply:
done